libraries/helpers.rb

```diff
@@ -127,8 +127,8 @@ def update_user_sql(new_resource)
       psql_command_string(new_resource, sql)
     end
 
-    def update_user_with_attributes_sql(new_resource, value)
-      sql = %(ALTER ROLE '#{new_resource.create_user}' SET #{attr} = #{value})
+    def update_user_with_attributes_sql(new_resource, attr, value)
+      sql = %(ALTER ROLE #{new_resource.create_user} SET #{attr} = #{value})
```

Comment:
Question about this change, (not a postgresql user) is it possible to create a user with a space in the name?

Reply:
I don't think you can (or at least, it's not the smartest idea). You can quote names btw, but then you have to use double quotes, single quotes a syntax error.

I defaulted to the style of other methods in the helper file, assuming that made most sense.